pootle/core/cache.py

```diff
@@ -43,3 +47,14 @@ def make_key(*args, **kwargs):
     return ':'.join([
         '%s=%s' % (k, v) for k, v in sorted(kwargs.iteritems())
     ])
+
+
+def get_cache(cache=None):
+    """Return a `cache` object named `cache` in none is specified or if the
+    request cache does not exist then the default cache is returned."""
+    if cache is None:
```

Comment:
Calling `caches[None]` raises `InvalidCacheBackendError` too, so you can reduce the whole function to the `try/except` block.